packages/commons-server/src/libs/templating-helpers/request-helpers.ts

```diff
@@ -24,6 +24,17 @@ export const RequestHelpers = function (
   environment: Environment
 ) {
   return {
+    // set status code
+    status: function (status: any) {
+      const intCode = parseInt(status, 10);
+      if (isNaN(intCode)) {
+        return;
+      }
+      if (!request.locals) {
+        request.locals = new Map<string, any>();
+      }
+      request.locals.set('statusCode', intCode);
+    },
```

Comment:
As this is the first helper dealing with the response, I would prefer to put this in the helpers.ts file or in a new response-helpers.ts file.

Reply:
We need to move also the test in a separate file?

Reply:
yes I tried to matche the lib/test filenames

Reply:
so I wait for your change

Reply:
Not sure I understand what you mean. Do you want me to move the code in a new file? Let me know and I will do it.

Reply:
Yes, if it is needed and you can do it is great, tnx